src/shapes/SurfaceSector.js

```diff
@@ -113,5 +113,29 @@ define([
             this._boundaries[3] = new Location(sector.minLatitude, sector.maxLongitude);
         };
 
+        // Internal use only. Intentionally not documented.
+        SurfaceSector.prototype.getReferencePosition = function () {
+            // Assign the first position as the reference position.
+            if (this._boundaries.length > 0){
```

Comment:
The check, if any, should be on the existence of the sector that is used afterwards. Are there cases where this sector can be `null` or `undefined`?